cmd/version.go

```diff
@@ -22,8 +22,8 @@ import (
 
 	log "sigs.k8s.io/etcdadm/pkg/logrus"
 
-	"github.com/ghodss/yaml"
 	"github.com/spf13/cobra"
+	"sigs.k8s.io/yaml"
```

Comment:
I agree with this one, but technically this might be a breaking change, because it changes the recognized tags from `yaml` to `json`.  Can we split this one into a separate PR?

Reply:
Ooops, you corrected me in office hours, I was confused :-)